### DISCUSSION TITLE
[아이디어] 2기 때, 애들이 많이 들어오려면 어떻게 해야할까요?

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#26] [03_아이디어]
여러분, Hexaton Class 2기를 여름방학 때 할 예정인데, 사람들을 많이 끌어오려면 어떻게 해야할까요?

여러분들의 의견이 듣고 싶네용...

**Comment by ghost** [comment]
저희가 잘 따라붙어서 좋은 결과 나오는게 우선이고 음... 수업의 장점을 강조하는것도 필요하지만 코딩을 할 줄 알면 어떤 점이 유리한가에 대하여 어필하는 것도 중요할 것 같아요
